src/CodeCompliance/Domain/Checks/Filters/IgnoreListFilter.php

```diff
@@ -68,6 +68,7 @@ public function getIgnoreSuffixes(): array
     {
         return [
             '/\/Kernel\//',
+            '/\/Shared\//',
```

Comment:
Can you please remind why we added `Shared` into the ignore list?

Reply:
For ignore TwigFunctionProvider, such as Spryker\Shared\Twig\TwigFunctionProvider

Reply:
Can we filter for the specific class but not the whole namespace.

Reply:
Changed to dedicated class '/^Spryker\/Shared\/Twig\/Twig\/TwigFunctionProvider$/',

Reply:
In this case should we remove `Shared`?

Reply:
Thanks, Shared is removed  https://github.com/spryker-sdk/upgrader/pull/18/commits/54d41640341f76b97d90d9a4e5f903e0eaacda8a